### DISCUSSION TITLE
GOODF

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#28] [General]
Dear Lars,

As you see, I have delved into your software body and soul and taking it very seriously. I would like this to be the best software ever written if it were possible - at least in terms of Virtual Organ Building. If I had any idea of programming, I would have created this software many years ago. At least we are getting it now - better late than never (I had given up hope that it would ever happen). My heartfelt thanks to you, Lars! You're a genius through and through!

You will make my greatest wish come true if the graphics turn out along the lines of jOrgan (Sven Meiers).... then we can say this GOODF is getting to the top-of-the-charts.  Add sound to enable voicing (along the lines of the commercial software Hauptwerk) and we will truly have the best ever software yet for one of the greatest, free virtual organ software - GrandOrgue.

I am hopeful as I am liking what I see and what I am getting. A few bugs..... yes.... but it would expected I guess. It must be a complex task creating this gem.

Please feel free to liaise with me to discuss and test things. We can do video conferences if you wish to trash out ideas and help clarify issues. I would be only too happy to do so.

**Comment by larspalo** [comment]
Thanks for testing GOODF! I hope that it should be useful already as it is, and that it can evolve even more in the future!
